config/test/kustomization.yaml

```diff
@@ -1,7 +1,9 @@
-bases:
-- ../default
-
-patchesStrategicMerge:
-- manager_image_patch.yaml
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
+patchesStrategicMerge:
+- manager_image_patch.yaml
+resources:
+- ../default
+images:
+- name: quay.io/kubevirt/kubemacpool
```

Comment:
Hey I think we need to take the image from local registry because its the dev environment manifest (config/test)
if we take the image from quay the changes we do and apply with cluster-sync will have no affect.

Reply:
Ups I have break this, let me fix, problem is that I did not fix it well here, that every time you run with external kustomize change this (I have use kustomize edit to change images) maybe we have to use other mechanism.